File: go/utility/feature_add.go
package utility

import "github.com/voxgig-sdk/arcgis-hub-world-countries-generalized-sdk/core"

func featureAddUtil(ctx *core.Context, f core.Feature) {
	client := ctx.Client
	features := client.Features

	client.Features = append(features, f)
}

File: php/utility/FeatureAdd.php
<?php
declare(strict_types=1);

// ArcgisHubWorldCountriesGeneralized SDK utility: feature_add

class ArcgisHubWorldCountriesGeneralizedFeatureAdd
{
    public static function call(ArcgisHubWorldCountriesGeneralizedContext $ctx, mixed $f): void
    {
        $ctx->client->features[] = $f;
    }
}
